operator-actions-framework/Makefile

```
@@ -0,0 +1,91 @@
+##@ Tools
+
+.PHONY: golangci-lint
+golangci-lint: $(GOLANGCI_LINT) ## Download golangci-lint locally if necessary.
+$(GOLANGCI_LINT): $(LOCALBIN)
+	$(call go-install-tool,$(GOLANGCI_LINT),github.com/golangci/golangci-lint/v2/cmd/golangci-lint,$(GOLANGCI_LINT_VERSION))
+
+$(LOCALBIN):
+	mkdir -p $(LOCALBIN)
+
+.PHONY: clean
+clean: ## Remove build artifacts.
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)